src/renderer/components/+workspaces/workspaces.tsx

```diff
@@ -41,10 +42,12 @@ export class Workspaces extends React.Component {
 
   saveWorkspace = (id: WorkspaceId) => {
     const draft = toJS(this.editingWorkspaces.get(id));
-    if (draft) {
+    const { error } = workspaceStore.saveWorkspace(draft);
+    if (!error) {
```

Comment:
Re: above; this would change to `const res = ...; if ("workspace" in res) { ...`